.appveyor.yml

```diff
@@ -11,6 +11,11 @@ environment:
       TOOLS: yes
       TARGET: release_debug
       ARCH: amd64
+    - VS: C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build\vcvarsall.bat
```

Comment:
It's Community: https://github.com/GodotBuilder/godot-builds/blob/master/appveyor.yml#L15

Reply:
Also, it's better to remove the VS 2015 build, AppVeyor is too slow (only one concurrent build job) and we're already waiting too long for PRs to be checked.

Reply:
Or actually no, it might be better to just stick to VS 2015. If it builds on 2015, I expect it should build on 2017 (which is less likely the other way around).

Reply:
NP. Should I close this or fix the Communirty bug?

Reply:
I'd say let's close it, it doesn't add much value for us now to have VS 2017 builds on top of the 2015 ones, and the latter are more limiting so better for testing builds IMO.